onevizion/task.py

```diff
@@ -63,8 +63,14 @@ def read(self, taskId = None, workplanId=None, orderNumber=None):
 				TraceMessage("Errors:\n{Errors}".format(Errors=json.dumps(self.OVCall.errors,indent=2)),0,TraceTag+"-Errors")
 			onevizion.Config["Error"]=True
 
+	def update(self, method='PUT', taskId, fields={}, dynamicDates=[]):
```

Comment:
propose to create a new public method for ```updatePartial``` or ```updatePartially```, return old method ```def update(self, taskId, fields={}, dynamicDates=[]):```, make this method private and invoke this private method from both of them.

Reply:
Okay, I agree. Fixed it.